lib/senkyoshi/models/qti.rb

```diff
@@ -10,15 +10,16 @@
 }.freeze
 
 module Senkyoshi
-  class QTI < Resource
-    def self.from(data, pre_data)
+  class QTI < RootResource
+    def self.from(data, pre_data, _ = nil)
```

Comment:
I think that depends on the use case though. If you want to be able to call `QTI.from` with either 2 or 3 arguments, you need the ` = nil`.